src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -512,7 +513,8 @@ writeDocx opts doc@(Pandoc meta _) = do
   let docPropsEntry = toEntry docPropsPath epochtime $ renderXml docProps
 
   let customProperties :: [(String, String)]
-      customProperties = [] -- FIXME
+      customProperties = [(k, escapeStringForXML $ lookupMetaString k meta) | k <- M.keys (unMeta meta)
```

Comment:
You shouldn't need escapeStringForXML here; I believe mknode will handle that automatically.
In fact, my guess is that with this you'll get double escaping.  Either way, you should test to make sure the escaping is being done correctly.

Reply:
Ok, I'll check that out, thanks for the review!

Besides manual testing, if I wanted to automate some tests, I guess I'd need to implement also reading the docx properties back into pandoc, right? (or is that already supported?)